app/frontend/src/cljs/teet/asset/asset_library_view.cljs

```diff
@@ -85,47 +103,106 @@
       [typography/SmallGrayText
        (tr [:asset :type-library :component-quantity-unit] {:unit quantity-unit})])
     [attribute-table open attributes]
-    (when (seq child-ctypes)
-      [:div.child-ctypes
-       (doall
-        (for [ct child-ctypes]
-          ^{:key (str (:db/id ct))}
-          [ctype open ct]))])]])
+    [child-links child-ctypes]]])
 
 (defn- fclass [open {attributes :attribute/_parent :as fclass}]
-  [collapsible open fclass
-   (str (tr [:asset :type-library :fclass]) " " (tr* fclass))
+  [:<>
+   [typography/Heading3
+    (str (tr [:asset :type-library :fclass]) " " (tr* fclass))]
    [:div
     (tr* fclass :asset-schema/description)
     [attribute-table open attributes]
-    (doall
-     (for [ct (:ctype/_parent fclass)]
-       ^{:key (str (:db/id ct))}
-       [ctype open ct]))]])
+    [child-links (:ctype/_parent fclass)]]])
 
 (defn- fgroup [open fgroup]
-  [collapsible open fgroup
-   (str (tr [:asset :type-library :fgroup]) " " (tr* fgroup))
+  [:<>
+   [typography/Heading3
+    (str (tr [:asset :type-library :fgroup]) " " (tr* fgroup))]
    [:div
     (tr* fgroup :asset-schema/description)
-    (doall
-     (for [fc (:fclass/_fgroup fgroup)]
-       ^{:key (str (:db/id fc))}
-       [fclass open fc]))]])
+    [child-links (:fclass/_fgroup fgroup)]]])
+
+(defn- rotl-type-badge-style []
+  {:font-size "70%"
+   :font-variant :all-small-caps
+   :margin-left "1rem"
+   :padding "0px 5px 0px 5px"
+   :border-radius "10px"
+   :display :inline-block
+   :background theme-colors/gray-lightest})
```

Comment:
Wonder if the text style defined here was one of those defined in common-styles.

Reply:
I couldn't find it, we have some badge styles in notification view but it's different.